hadoop-ozone/dist/src/main/compose/ozonesecure/test.sh

```diff
@@ -27,39 +27,42 @@ export SECURITY_ENABLED=true
 
 : ${OZONE_BUCKET_KEY_NAME:=key1}
 
-start_docker_env
+for enable in true false; do
```

Comment:
This trick doubled the execution time of the secure acceptance test which was already 50 min. It increased the full build time with ~50 mins and (because the parallel execution assumed all the tests finish in 1 hour) the full build is also slower with 30-40 minutes. 

Do we really need this trick?

(By the way the `enable` variable name is not very expressive, and it should be added to the `-N` because different tests will overwrite the results of each other) 

cc @adoroszlai @bharatviswa504

Reply:
Thanks @elek It is added to test in a stand alone SCM cluster with ratis enable/disable.
As some of the integration issues were missed to identify in UTs. Later we have added a new acceptance test secure-ha, which tests end-to-end HA. (It would be good to have, but if we think it is expensive or any other way we can run this it would be good)
I think now we can remove this if we think this is causing a longer CI duration. Thanks, @adoroszlai for opening a Jira to fix this.

Reply:
Thanks to explain it @bharatviswa504. I think it's too expensive to run ALL the tests in two modes. For example, we can assume that the linked and real buckets working at the same way as they already tested. We don't need to test all of them in both cases (IMHO).

I think it's fine to test different settings to make sure if it works. But we can adjust the required tests. For the main/default case we can execute all the tests and the other mode we can execute only a minimum set.